[2024-04-27 23:19:33,880] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: exampleSimon.runme_0 2024-04-26T00:00:00+00:00 [queued]>
[2024-04-27 23:19:33,997] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: exampleSimon.runme_0 2024-04-26T00:00:00+00:00 [queued]>
[2024-04-27 23:19:33,999] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2024-04-27 23:19:34,002] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2024-04-27 23:19:34,003] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2024-04-27 23:19:50,680] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): runme_0> on 2024-04-26T00:00:00+00:00
[2024-04-27 23:19:50,709] {standard_task_runner.py:52} INFO - Started process 224 to run task
[2024-04-27 23:19:50,887] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'exampleSimon', 'runme_0', '2024-04-26T00:00:00+00:00', '--job-id', '93', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/helloAgain.py', '--cfg-path', '/tmp/tmpenyphqt5', '--error-file', '/tmp/tmpm7tdlvmi']
[2024-04-27 23:19:50,936] {standard_task_runner.py:77} INFO - Job 93: Subtask runme_0
[2024-04-27 23:19:51,870] {logging_mixin.py:104} INFO - Running <TaskInstance: exampleSimon.runme_0 2024-04-26T00:00:00+00:00 [running]> on host 10d8235a57dc
[2024-04-27 23:19:52,306] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=exampleSimon
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2024-04-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-26T00:00:00+00:00
[2024-04-27 23:19:52,323] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-04-27 23:19:52,330] {bash.py:158} INFO - Running command: echo "exampleSimon__runme_0__20240426" && sleep 1
[2024-04-27 23:19:52,374] {bash.py:169} INFO - Output:
[2024-04-27 23:19:52,376] {bash.py:173} INFO - exampleSimon__runme_0__20240426
[2024-04-27 23:19:53,375] {bash.py:177} INFO - Command exited with return code 0
[2024-04-27 23:19:53,533] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=exampleSimon, task_id=runme_0, execution_date=20240426T000000, start_date=20240427T231933, end_date=20240427T231953
[2024-04-27 23:19:53,784] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-27 23:19:53,914] {local_task_job.py:146} INFO - Task exited with return code 0
